src/tools.py

```diff
@@ -52,10 +52,7 @@ def tools_versions():
 
 
 def tools_rootpw(new_password, check_strength=True):
-    import spwd
-
     from yunohost.utils.password import (
-        _hash_user_password,
```

Comment:
Hmmmm but there are other places where `_hash_user_password` is used, shouldn't we fix that function rather than just `tools_rootpw` ?

Reply:
(Ah I just realized the new code calls passwd, it doesn't hash the password per se)